_posts/2021-03-15-elastic-training.md

```diff
@@ -0,0 +1,159 @@
+---
+toc: true
+layout: post
+categories: [elastic training, operators]
+title: "Elastic Training with MPI Operator and Practise"
+comments: true
+author: "<a href='https://www.linkedin.com/in/gaocegege/'>Ce Gao</a>, <a href='https://www.linkedin.com/in/wang-zhang'>Wang Zhang</a>"
+---
+ 
+With increase in the size of dataset and deep learning models, distributed training emerges as the mainstream approach for training neural network models in industry. While it is feasible now to launch a massive distributed training job on Kubernetes with Kubeflow, advanced features like elastic workload and other cost mitigation approaches remain leashed when we talk about deep learning jobs on Kubernetes.
+
+To address issues on cost and resource utilization, the [TKE (Tencent Kubernetes Engine)](https://intl.cloud.tencent.com/product/tke) AI team designs and implements **Elastic Training** in Kubeflow community.
+ 
+Here we present how the elastic training is performed on Kubernetes. Validated with experiments under circumstances, elastic training lowers cost for distributed training on cloud.
+ 
+## Background
+ 
+Let's first recap training deep learning models. When we talk about 'training', it refers generally to iteratively optimizing parameters in a neural network model with its gradient descent. Accelerated with GPUs, the training can speed up for 10-100 times.
```

Comment:
`training`? change ' to `

Reply:
single quotation seems fine as it's not referring code script.